### ISSUE TITLE
Not compatible with PhantomJS 1.x

### ISSUE THREAD

**Issue by davidgoli** [justin-lau/ember-intl-tel-input#4]
``` es6
ember-intl-tel-input/addon/components/intl-tel-input.js:    var notifyPropertyChange = this.notifyPropertyChange.bind(this, 'value');
```

`Function.prototype.bind` is not defined in Phantom JS 1.x: https://github.com/ariya/phantomjs/issues/10522

So, this fails when using eg. Travis CI test runners.

**Comment by justin-lau** [comment]
PhantomJS 2.0 has been out for quite a while. It has already addressed the `Function.prototype.bind` problem. I do not see a reason to support 1.x anymore.

If you insist to use PhantomJS 1.x, you could use any workarounds available on the issue you referenced.

**Comment by davidgoli** [comment]
@justin-lau That's fair, and this is your repo, but Travis CI still doesn't support PhantomJS 2.0 out of the box. This is the only addon we're using in our Ember project (including Ember itself) that doesn't support PhantomJS 1.9, and the only problem we've had with builds on Travis CI, and the provided PR fixes the only incompatibility you have with 1.9. It's a trivial fix, but if you aren't interested I'll just keep pointing at my fork. Thanks!

**Comment by justin-lau** [comment]
Thank you for understanding. A more common approach would be adding a polyfill at the beginning of the test scripts. This way you don't have to use a forked repository, and you make sure any addons you add in the future won't cause the same problem.
